ui/app/src/modules/Preview/QuickCreate.tsx

```diff
@@ -80,71 +81,71 @@ const useStyles = makeStyles((theme: Theme) =>
   })
 );
 
-export default function QuickCreate() {
+export function QuickCreateMenu({ anchorEl, onMenuClose, previewItem }) {
```

Comment:
Create interface and define props and types

Reply:
Done.